pyramid_oereb/standard/sources/plr.py

```diff
@@ -271,6 +271,8 @@ def from_db_to_document_records(self, legal_provisions_from_db, article_numbers=
                 )
             )
             document_records.append(self._documents_record_class(
+                # FIXME figure out which value to use
```

Comment:
Check in print on how to figure this out.

Reply:
See: https://github.com/camptocamp/pyramid_oereb/blob/bc1101b4e100bdd764f9a4b83023eee41f0ec5c4/pyramid_oereb/contrib/print_proxy/mapfish_print.py#L237-L266

I have the feeling that references and articles are always empty in the current print implementation...